apps/src/templates/instructions/ReviewTab.jsx

```diff
@@ -177,6 +178,11 @@ class ReviewTab extends Component {
           comments: comments,
           forceRecreateEditorKey: this.state.forceRecreateEditorKey + 1
         });
+      })
+      .fail(result => {
+        if (result.status === 404) {
+          this.setState({authorizationError: true});
+        }
```

Comment:
If we fail for another reason (like a 500) do we want to show an error message as well?

Reply:
that's coming as a follow up to this :)